bottlerocket/testsys/src/run_aws_k8s.rs

```diff
@@ -102,6 +102,10 @@ pub(crate) struct RunAwsK8s {
     #[structopt(long)]
     cluster_provider_pull_secret: Option<String>,
 
+    /// Path to eksctl config file.
+    #[structopt(long)]
+    cluster_config_path: Option<String>,
```

Comment:
To keep it consistent with `sonobuoy_e2e_repo_config`
```suggestion
    cluster_config: Option<String>,
```